stylesheets/_component.datatables.scss

```diff
@@ -83,6 +83,8 @@ a.dt-button {
 
     td,
     th {
+        overflow: hidden;
```

Comment:
DT does some funky stuff when scrolling is enabled, the `thead` and `tbody` are moved into tables entirely separate from each other and the column widths are calculated by DT javascript. The existing `thead` is hidden from from view, but the sortable arrows were still visible. This hides them.